access/src/main/webapp/WEB-INF/jsp/searchResults/resultsList.jsp

```diff
@@ -18,17 +18,21 @@
 <%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
 <%@ page trimDirectiveWhitespaces="true" %>
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
-<div class="bottomline paddedline">
-	<c:import var="navigationBar" url="searchResults/navigationBar.jsp">
-		<c:param name="queryMethod">${queryMethod}</c:param>
-	</c:import>
-	${navigationBar}
-	<c:import url="searchResults/sortForm.jsp">
-		<c:param name="queryMethod">${queryMethod}</c:param>
-		<c:param name="currentSort">${searchState.sortType}</c:param>
-		<c:param name="currentSortOrder">${searchState.sortNormalOrder}</c:param>
-	</c:import>
-</div>
+
+<c:if test="${empty param.collectionBrowse}">
```

Comment:
How about renaming this `param.excludeNavigationBar` or something similar to make it clear what the behavior is? This seems broader than the collectionBrowse.